Dalamud/Game/Inventory/GameInventory.cs

```diff
@@ -403,6 +403,9 @@ internal class GameInventoryPluginScoped : IInternalDisposableService, IGameInve
     /// <inheritdoc/>
     public event IGameInventory.InventoryChangedDelegate<InventoryItemMergedArgs>? ItemMergedExplicit;
 
+    /// <inheritdoc/>
+    public ReadOnlySpan<GameInventoryItem> GetInventoryItems(GameInventoryType type) => GameInventoryItem.GetReadOnlySpanOfInventory(type);
```

Comment:
How is it that this use case hasn't come up yet?

Reply:
Because everyone that needs inventory items uses `InventoryManager` directly? I don't know. 🤷‍♂️